packages/cli/plugin-tailwind/src/tailwind.ts

```diff
@@ -52,19 +39,11 @@ const getTailwindConfig = (
 
   const designSystemConfig = getPureDesignSystemConfig(designSystem ?? {});
 
-  const [exist, key] = checkIfExistNotAllowKeys(tailwindConfig);
-
-  if (exist) {
-    logger.error(
-      `should not exist '${key}' on tools.tailwindcss, please remove it`,
-    );
-    // eslint-disable-next-line no-process-exit
-    process.exit(0);
```

Comment:
why remove this check action

Reply:
To make `tools.tailwindcss` same as the native Tailwind CSS config, so users do not need to delete `theme` option when using `tools.tailwindcss`.